src/components/about/Mission.js

```diff
@@ -1,7 +1,26 @@
 import React from "react";
+import Image from "next/image";
 
-const mission = () => {
-  return <div>mission</div>;
+const Mission = ({ title, img, texts }) => {
```

Comment:
look at Figma carefully and make sure to have the correct fonts for each set of text